plugin-server/src/main/ingestion-queues/session-recording/blob-ingester/session-manager.ts

```diff
@@ -91,26 +92,33 @@ export class SessionManager {
             await this.addToChunks(message)
         }
 
-        await this.flushIfNecessary()
+        this.buffer.lastMessageReceivedAt = kafkaMessageTimestamp || Date.now()
```

Comment:
Why would this ever not be set?

Reply:
I was just passing the type along :) Since the caller doesn't guarantee it is set I dumped in a sensible default